src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -36,24 +49,32 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.concurrent.atomic.AtomicReference;
 
-import net.greghaines.jesque.Config;
-import net.greghaines.jesque.Job;
-import net.greghaines.jesque.JobFailure;
-import net.greghaines.jesque.WorkerStatus;
-import net.greghaines.jesque.json.ObjectMapperFactory;
-import net.greghaines.jesque.utils.JedisUtils;
-import net.greghaines.jesque.utils.JesqueUtils;
-import net.greghaines.jesque.utils.ScriptUtils;
-import net.greghaines.jesque.utils.VersionUtils;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import redis.clients.jedis.Jedis;
-import redis.clients.jedis.exceptions.JedisException;
-
-import com.fasterxml.jackson.core.JsonParseException;
-import com.fasterxml.jackson.databind.JsonMappingException;
+import static net.greghaines.jesque.utils.ResqueConstants.COLON;
+import static net.greghaines.jesque.utils.ResqueConstants.DATE_FORMAT;
+import static net.greghaines.jesque.utils.ResqueConstants.FAILED;
+import static net.greghaines.jesque.utils.ResqueConstants.INFLIGHT;
+import static net.greghaines.jesque.utils.ResqueConstants.JAVA_DYNAMIC_QUEUES;
+import static net.greghaines.jesque.utils.ResqueConstants.PROCESSED;
+import static net.greghaines.jesque.utils.ResqueConstants.QUEUE;
+import static net.greghaines.jesque.utils.ResqueConstants.QUEUES;
+import static net.greghaines.jesque.utils.ResqueConstants.STARTED;
+import static net.greghaines.jesque.utils.ResqueConstants.STAT;
+import static net.greghaines.jesque.utils.ResqueConstants.WORKER;
+import static net.greghaines.jesque.utils.ResqueConstants.WORKERS;
+import static net.greghaines.jesque.worker.JobExecutor.State.NEW;
+import static net.greghaines.jesque.worker.JobExecutor.State.RUNNING;
+import static net.greghaines.jesque.worker.JobExecutor.State.SHUTDOWN;
+import static net.greghaines.jesque.worker.JobExecutor.State.SHUTDOWN_IMMEDIATE;
+import static net.greghaines.jesque.worker.WorkerEvent.JOB_EXECUTE;
+import static net.greghaines.jesque.worker.WorkerEvent.JOB_FAILURE;
+import static net.greghaines.jesque.worker.WorkerEvent.JOB_PROCESS;
+import static net.greghaines.jesque.worker.WorkerEvent.JOB_SUCCESS;
+import static net.greghaines.jesque.worker.WorkerEvent.WORKER_ERROR;
+import static net.greghaines.jesque.worker.WorkerEvent.WORKER_POLL;
+import static net.greghaines.jesque.worker.WorkerEvent.WORKER_START;
+import static net.greghaines.jesque.worker.WorkerEvent.WORKER_STOP;
+import static net.greghaines.jesque.worker.WorkerImpl.NextQueueStrategy.DRAIN_WHILE_MESSAGES_EXISTS;
+import static net.greghaines.jesque.worker.WorkerImpl.NextQueueStrategy.RESET_TO_HIGHEST_PRIORITY;
```

Comment:
Please keep the import order that the rest of the project uses:
<static imports> (If more than a few from the same class, import star)
<regular imports>